src/modules/auth/clients/FormFields.js

```diff
@@ -18,11 +25,13 @@ export function TextInput ({ name, register, labelName, disabled }) {
 				name={name}
 				type="text"
 				disabled={disabled}
+				required={name == "client_name" ? true : false}
 				onChange={reg.onChange}
 				onBlur={reg.onBlur}
 				innerRef={reg.ref}
-
+				invalid={(errors?.preferred_client_id) && errors.preferred_client_id}
```

Comment:
@Aringoaway `invalid={errors.preferred_client_id && errors.preferred_client_id}` should be enough

Reply:
@Pe5h4 I tested it and `?` here it is necessary => `invalid={errors?.preferred_client_id && errors.preferred_client_id}`